SimPEG/EM/Static/DC/SurveyDC.py

```diff
@@ -131,14 +131,14 @@ def getABMN_locations(self):
         self.m_locations = np.vstack(m_locations)
         self.n_locations = np.vstack(n_locations)
 
-    def drapeTopo(self, mesh, actind, option='top', topography=None):
+    def drapeTopo(self, mesh, actind, option='top', topography=None, force=False):
         if self.a_locations is None:
             self.getABMN_locations()
 
         # 2D
         if mesh.dim == 2:
             if self.survey_geometry == "surface":
-                if self.electrodes_info is None:
+                if self.electrodes_info is None or force:
```

Comment:
What does this force do?

Reply:
This is considering multiple times to set electrode locations, which may not be that much often, but possible